ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py

```diff
@@ -0,0 +1,13 @@
+import ivy
+
+
+def argsort(
```

Comment:
This is to be implemented according to the docs of the respective frontend function. Which states that the the arguments in the function definition should be according to the `numpy` docs of the respective function(https://numpy.org/doc/stable/reference/generated/numpy.argsort.html).
Moreover, the function should be implemented using `ivy`
I've also attached the link to the docs below.
Ivy frontend: https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html
Ivy frontend tests: https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html
Thanks! :)

Reply:
done